PythonToolbox/tests/test_api.py

```diff
@@ -11,126 +11,169 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from quantconnect import ApiConnection
 from quantconnect.api import Api
 from quantconnect.LeanReportCreator import LeanReportCreator
+from time import sleep
 import unittest
 import os
 
 class TestApi(unittest.TestCase):
 
     def setUp(self):
         # Please add your credentials. Found at https://www.quantconnect.com/account 
-        self.userId = ""
-        self.testToken = ""
+        self.userId = 0
+        self.testToken = ''
```

Comment:
Maybe should it try to load them from the config?

Reply:
Since we are not adding the config to the package, we are not using it. It would have to be copied because of its relative position.